docs/Tutorials/BeginnersTutorial.md

```diff
@@ -0,0 +1,547 @@
+can read [the paper](https://arxiv.org/abs/2109.11645) on the ArXiv.
+
+### Running the Kuzmin Problem
+
+To demonstrate DG+FD, we will be evolving the \link
+ScalarAdvection::Solutions::Kuzmin Kuzmin \endlink problem using the
+`EvolveScalarAdvectionKuzmin2D` executable. This is a simple test problem that
+rotates a set of geometric shapes with uniform angular velocity, which can be
+used to evaluate how well a numerical code can handle discontinuities stably
+over time. Inside the container build this executable and then link to it in a
+new directory `runs2` where you will run it. Also copy the default input file in
+`tests/InputFiles/ScalarAdvection/Kuzmin2D.yaml` to this new `runs2` directory.
+
+### Changing the Default Input File
+
+The default input file has very low resolution so we'll need to crank that up a
+bit. The way to do this is to change the initial refinement levels and initial
+number of grid points which are located in
+
+```yaml
+DomainCreator:
+  Rectangle:
+    ...
+    InitialRefinement: [x, y]
+    InitialGridPoints: [x, y]
+```
+
+`InitialRefinement:` represents how many times we split a `Block` in half in
+order to create `Element`s, which are the fundamental units of our domain. So an
+initial refinement of `[1, 1]` means we split a single Block into 4 elements
+(split in half once in each direction). For an initial refinement of `[2, 2]` we
+first do 1 refinement like before, and then split each of the resulting 4
+elements in half again in each direction, resulting in 16 total Elements. To
+determine the total number of Elements for a given refinement (same in all
+directions), just do \f$2^{\mathrm{Dim * Refinement}}\f$. If you're confused by
+the terminology we use to describe the domain, we have a \ref domain_concepts
+guide that explains all terms related to our domain.
+
+`InitialGridPoints` represents the number of grid points per dimension in each
+Element after the final refinement has been applied. So if we had an initial
+refinement of `[2, 2]` like above and then initial grid points `[3, 3]` in each
+Element, we'd have a total of 9x16=144 grid points.
+
+As for actual numbers to use, you can experiment to see what gives good,
+well-resolved results. You'll definitely need more refinement than the default
+input file, but since refinement scales exponentially, this can become very
+expensive very quickly. On a laptop, you probably shouldn't go higher than
+refinement `[6, 6]`. As for grid points, this will depend on how much refinement
+you have. If you have a ton of small elements, you won't need too many grid
+points to resolve the solution; something like `[4, 4]` would work. If you don't
+have a lot of refinement, you may want more grid points if you still want to
+resolve your solution. In SpECTRE, however, it is our philosophy that if you
+want to resolve your system, do it with more refinement rather than more grid
+points.
```

Comment:
It is?  I thought higher-order elements generally got better results.  That's what I see in Figure 4 of the paper you just cited.

Reply:
Yea I'd say either remove the sentence or say something like this:

> For a DG scheme, increasing the number of grid points (p refinement) reduces the numerical error exponentially where the solution is smooth, so computational resources are used more effectively. However, to resolve shocks and discontinuities we have to refine the domain into more and smaller elements instead (h refinement). Striking the most effective balance between h and p refinement in different parts of the domain is the job of an adaptive mesh refinement (AMR) algorithm.